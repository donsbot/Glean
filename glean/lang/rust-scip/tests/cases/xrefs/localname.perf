{ "@generated": null, "scip.SymbolName.1": 6 }