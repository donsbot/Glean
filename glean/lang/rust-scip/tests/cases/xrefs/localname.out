[
  "@generated",
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 (acc)" },
      "name": { "key": "(acc" }
    }
  },
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 (n)" },
      "name": { "key": "(n" }
    }
  },
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 f1_go()." },
      "name": { "key": "f1_go()" }
    }
  },
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 sum_f1()." },
      "name": { "key": "sum_f1()" }
    }
  },
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 sum_f2()." },
      "name": { "key": "sum_f2()" }
    }
  },
  {
    "key": {
      "symbol": { "key": "rust-analyzer cargo basics 0.1.0 sum_f3()." },
      "name": { "key": "sum_f3()" }
    }
  }
]